src/pages/base/Environment/index.jsx

```diff
@@ -23,7 +30,17 @@ const environmentStyle = environment => {
 };
 
 export const Environment = () => {
-  const { ENVIRONMENT: environment } = config;
-  const isShow = environment === 'local' || environment === 'development' || environment === 'staging';
-  return isShow ? <p css={environmentStyle(environment)}>{environment}</p> : null;
+  const { ENVIRONMENT: environmentTitle } = config;
+  switch (environmentTitle) {
+    case ENV.LOCAL || ENV.DEV:
```

Comment:
`ENV.LOCAL || ENV.DEV`의 값이 `ENV.LOCAL`이라서 `ENV.DEV`가 잡히지 않습니다.

```suggestion
    case ENV.LOCAL:
    case ENV.DEV:
```

Reply:
:parrot:

Reply:
수정완료했습니다.